boards/arm/arduino_nano_33_ble/doc/index.rst

```diff
@@ -97,21 +97,27 @@ You can get this variant of bossac with one of two ways:
 
    #. Install the board support package within the IDE
    #. Change your IDE preferences to provide verbose logging
-   #. Build and flash a sample application, and read the logs to figure out where Arduino stored bossac.
+   #. Build and flash a sample application, and read the logs to figure out where Arduino stored bossac. (In most Linux based systems the path is `$HOME/.arduino15/packages/arduino/tools/bossac/1.9.1-arduino2/bossac`, it may vary according to which version of Arduino IDE you have installed.)
```

Comment:
Please keep lines under 100 chars and use double-backtick for unformatted text instead of single backtick.

Reply:
okay then I will put this in the next line

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/38542/commits/90a1c68af2f7ef837043be7a597884d6f8990557

Reply:
I could squash these 2 commits if required

Reply:
Yes, please squash them.